src/workers/notifications/commentMention.ts

```diff
@@ -24,6 +24,10 @@ const worker: NotificationWorker = {
     if (!postCtx) {
       return;
     }
+    const authors = [postCtx.post.authorId, postCtx.post.scoutId];
+    if (authors.includes(data.commentMention.mentionedUserId)) {
+      return;
+    }
```

Comment:
Can't we simply use `new set[author, scout]`?
Meaning we could unique the array no?